frontend/src/main/scala/bloop/engine/Interpreter.scala

```diff
@@ -223,9 +229,66 @@ object Interpreter {
       case project :: Nil =>
         state.build.getProjectFor(project) match {
           case Some(project) =>
-            compileAnd(cmd, state, List(project), cmd.excludeRoot, "`console`")(
-              state => Tasks.console(state, project, cmd.excludeRoot)
-            )
+            compileAnd(cmd, state, List(project), cmd.excludeRoot, "`console`") { state =>
+              cmd.repl match {
+                case ScalacRepl =>
+                  if (cmd.ammoniteVersion.isDefined) {
+                    val errMsg =
+                      "Specifying an Ammonite version while using the Scalac console does not work"
+                    Task.now(state.withError(errMsg, ExitStatus.InvalidCommandLineOption))
+                  } else {
+                    Tasks.console(state, project, cmd.excludeRoot)
+                  }
+                case AmmoniteRepl =>
+                  // Look for version of scala instance in any of the projects
+                  def findScalaVersion(dag: Dag[Project]): Option[String] = {
+                    dag match {
+                      case Aggregate(dags) => dags.flatMap(findScalaVersion).headOption
+                      case Leaf(value) => value.scalaInstance.map(_.version)
+                      case Parent(value, children) =>
+                        children.flatMap(findScalaVersion) match {
+                          case Nil => value.scalaInstance.map(_.version)
+                          case xs => Some(xs.head)
+                        }
+                    }
+                  }
+
+                  import bloop.engine.ExecutionContext.ioScheduler
+                  val dag = state.build.getDagFor(project)
+                  // If none version is found (e.g. all Java projects), use Bloop's scala version
+                  val scalaVersion = findScalaVersion(dag)
+                    .getOrElse(ScalaInstance.scalaInstanceFromBloop(state.logger))
+
+                  val ammVersion = cmd.ammoniteVersion.getOrElse("latest.release")
+                  val coursierCmd = List(
+                    "coursier",
+                    "launch",
+                    s"com.lihaoyi:ammonite_$scalaVersion:$ammVersion",
+                    "--main-class",
+                    "ammonite.Main",
+                    "--scala-version",
```

Comment:
Is this the place where ammonite options are specified? If so, can you please add `--no-remote-logging` here? See https://github.com/lihaoyi/Ammonite/issues/607 for context.

Reply:
No, I won't enable that ammonite flag by default in Bloop. If you're interested in a workaround, feel free to ping me in Gitter and we can discuss a way people interested in disabling that option can make a pull request and we can merge it.

Reply:
The reason why I suggested a change in the default configuration is because some users might be unknowingly sharing their info to a third-party.

If I have to manually change user configuration I will simply choose to use the alternate shell (scalac) and avoid the problem entirely.